### PR TITLE
CVSL-622 accessibility statement page added

### PR DESCRIPTION
**Pull request by GurnankCheema**
Page accessed from new Accessibility statement link in footer

Screenshot 2022-09-01 at 15 55 14

### CODE REVIEW

File: server/views/pages/accessibilityStatement.njk [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{% extends "layout.njk" %}
+
+{% set pageTitle = applicationName + " accessibility statement" %}
+{% set pageId = 'accessibility-statement-page' %}
+{% set cvlLink = "https://create-and-vary-a-licence.hmpps.service.justice.gov.uk" %}
```
Comment (andrewrlee) [inline server/views/pages/accessibilityStatement.njk:5]: this should probably be `config.domain`, so it varies per environment - you could add it to `app.locals`?

Comment (andrewrlee) [reply server/views/pages/accessibilityStatement.njk:5]: or pass it through the render 🤷‍♂️

Comment (GurnankCheema) [reply server/views/pages/accessibilityStatement.njk]: done

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**